.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+## Status
+**READY/IN DEVELOPMENT/HOLD**
```

Comment:
I think GitHub now as "draft" PRs so maybe we just use those for non-ready PRs and remove this?